src/booleans.js

```diff
@@ -11,6 +11,8 @@ const either = (a, b) => {
 };
 
 const none = (a, b) => {
+  // you can just return the expression.
+  // also code on new lines
```

Comment:
Closing curly braces for code blocks should always be on new lines, and opening curlies should always trail:

```js
if (a === false && b === false) {
  return true
} else {
  return false
}
```

Again, you could omit the if / else altogether here.